pages/learn/manage/configuration.md

```diff
@@ -17,7 +17,6 @@ This list contains configuration variables that can be used with {{ $names.compa
 Name | Default | Description | Valid from
 --- | --- | --- | ---
 RESIN_SUPERVISOR_CONNECTIVITY_CHECK | true | Enable / Disable VPN connectivity check | v1.3.0
-RESIN_SUPERVISOR_DELTA | false | Enable / Disable [delta updates][deltas] | v1.7.0
```

Comment:
be sure to drop the footnote link if it's unused too

Reply:
godo catch, updated. I also missed some image files but those are removed now too.